router/wrap_unwrap.gno

```diff
@@ -0,0 +1,46 @@
+package router
+
+import (
+	"std"
+
+	"gno.land/r/demo/wugnot"
+
+	"gno.land/p/demo/ufmt"
+	"gno.land/r/demo/consts"
+)
+
+func wrap(ugnotAmount uint64) {
+	if ugnotAmount <= 0 {
+		panic(ufmt.Sprintf("cannot wrap 0 ugnot"))
+	}
+
+	if ugnotAmount < consts.UGNOT_MINT_DEPOSIT_TO_WRAP {
+		panic(ufmt.Sprintf("Deposit below minimum: %d/%d ugnot.", ugnotAmount, consts.UGNOT_MINT_DEPOSIT_TO_WRAP))
+	}
+
+	// WRAP IT
+	wugnotAddr := std.DerivePkgAddr(consts.WRAPPED_WUGNOT)
+	banker := std.GetBanker(std.BankerTypeRealmSend)
+	banker.SendCoins(consts.ROUTER_ADDR, wugnotAddr, std.Coins{{"ugnot", int64(ugnotAmount)}})
+	wugnot.Deposit() // ROUTER HAS WUGNOT
+
+	// SEND WUGNOT: ROUTER -> USER
+	wugnot.Transfer(a2u(std.GetOrigCaller()), ugnotAmount)
+}
+
+func unwrap(wugnotAmount uint64) {
+	if wugnotAmount <= 0 {
+		return
+	}
```

Comment:
```suggestion
	if wugnotAmount == 0 {
		return
	}
```

Since `wugnotAmount` is of type `uint64`, I think it seems unnecessary to check fr value is less than zero. Moreover, it's not specified what action should be taken. Is it trigger a panic if the amount falls into this condition?

Reply:
it panics in `wugnot` contract, so just checking `==0` is enough